packages/slice-machine/src/domain/customType.ts

```diff
@@ -537,8 +560,8 @@ export function reorderFields<T>(args: ReorderFieldsArgs<T>) {
   const fieldEntries = Object.entries(fields);
   const [removedEntry] = fieldEntries.splice(sourceIndex, 1);
   fieldEntries.splice(destinationIndex, 0, removedEntry);
-  const reorderedFields = Object.fromEntries(fieldEntries);
 
+  const reorderedFields = Object.fromEntries(fieldEntries);
```

Comment:
💡 Most of the time we recommend having a blank line before return, for example: https://eslint.org/docs/latest/rules/newline-before-return

Reply:
fixed